scripts/testnet.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# API used for funding the calls, ideally https://api.hoprnet.org
+declare API_ENDPOINT="https://api-hopr-oxbs3v49y-jjperezaguinaga.vercel.app"
```

Comment:
Please provide pointer to GH repo here as well, so newcomers may check out the API without having to search too long.

Reply:
Solved in [b5c207e](https://github.com/hoprnet/hoprnet/pull/3595/commits/b5c207e04e42c5d79b7153ccfbe625d1097ed450)